index.js

```diff
@@ -5,7 +5,10 @@ const GENERATED_DIR = 'generated'
 const LISTS_DIR = 'exception-lists'
 
 const defaultConfig = JSON.parse(fs.readFileSync('default-config.json'))
-defaultConfig.version = Date.now()
+
+if (process.env.USING_CI) {
```

Comment:
Is this needed?

1. Are we envisioning this to still be run by config authors?
2. Should we remove the `generated/` directory from the repo and also add it to the `.gitignore`?

Reply:
This is up for debate. When we previously ran the build script in the action we had the issue that a maintainer would:
- generate locally creating one timestamp
- Merge their PR which builds and generates another timestamp

This would cause confusion because the timestamp in the repo was different than the published version.

I'm personally in favor of keeping the generated files in the repo as a quick reference of what's in prod however @kdzwinel also voiced that we could ignore these files in the repo and only build in the actions and just use the online versions for reference (possibly allowing s3 to list the directory their in).

Reply:
I think if we want full visibility of published code we could have the CI create a PR to a different repo.

The rationale for removing the generated changes is that it'll reduce the amount of merge conflicts and noise for a reviewer.

Perhaps we can mark the generated files as binaries? Would this reduce the noise in github / git?

Reply:
Uuu… it'd be fancy to release generated files to https://github.com/duckduckgo/privacy-configuration/releases . This action seems to do that: https://github.com/marvinpinto/action-automatic-releases

Reply:
Wish granted 🪄